jdaviz/configs/imviz/plugins/links_control/links_control.py

```diff
@@ -12,39 +12,24 @@
 class LinksControl(TemplateMixin):
     template = load_template("links_control.vue", __file__).tag(sync=True)
 
-    # TODO: Use radio button groups?
     link_types = List(['Pixels', 'WCS']).tag(sync=True)
-    wcs_fallback_schemes = List(['None', 'Pixels']).tag(sync=True)
-    wcs_use_affine_options = List(['Yes', 'No']).tag(sync=True)
+
+    # default states. NOTE: same case as options above, any necessary casting
+    # to internal API formats should be in vue_do_link)
+    link_type = Unicode('Pixels').tag(sync=True)
+    wcs_use_fallback = Bool(True).tag(sync=True)
+    wcs_use_affine = Bool(True).tag(sync=True)
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._link_type = 'pixels'
-        self._wcs_fallback_scheme = None
-        self._wcs_use_affine = True
-
-    def vue_link_type_selected(self, event):
-        if event == 'WCS':
-            self._link_type = 'wcs'
-        else:
-            self._link_type = 'pixels'
-
-    def vue_wcs_fallback_scheme_selected(self, event):
-        if event == 'None':
-            self._wcs_fallback_scheme = None
-        else:
-            self._wcs_fallback_scheme = 'pixels'
-
-    def vue_affine_option_selected(self, event):
-        if event == 'Yes':
-            self._wcs_use_affine = True
-        else:
-            self._wcs_use_affine = False
 
     def vue_do_link(self, *args, **kwargs):
         """Run :meth:`jdaviz.configs.imviz.helper.Imviz.link_data`

```

Comment:
Ops... I missed this one.
```suggestion
        """Run :func:`jdaviz.configs.imviz.helper.link_image_data`

```